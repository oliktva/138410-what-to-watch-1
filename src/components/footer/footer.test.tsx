import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

import Footer from './footer';

Enzyme.configure({adapter: new Adapter()});

it(`renders correctly`, () => {
  const footer = shallow(<Footer />);

  expect(toJson(footer)).toMatchSnapshot();
});
