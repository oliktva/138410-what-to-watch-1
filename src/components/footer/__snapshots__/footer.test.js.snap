// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="page-footer"
>
  <Logo
    isLight={true}
  />
  <div
    className="copyright"
  >
    <p>
      Â© 2019 What to watch Ltd.
    </p>
  </div>
</footer>
`;
